libs/application/templates/parental-leave/src/forms/OtherParentApproval.ts

```diff
@@ -7,29 +7,70 @@ import {
   Form,
   FormModes,
   coreMessages,
+  buildTextField,
+  Application,
 } from '@island.is/application/core'
 
 import Logo from '../assets/Logo'
+import { YES } from '../constants'
 import { otherParentApprovalFormMessages } from '../lib/messages'
+import { getApplicationAnswers } from '../parentalLeaveUtils'
 
 export const OtherParentApproval: Form = buildForm({
   id: 'OtherParentApprovalForParentalLeave',
-  title: otherParentApprovalFormMessages.formTitle,
+  // title: otherParentApprovalFormMessages.formTitle,
+  title: 'Tilfærsla réttinda',
   logo: Logo,
   mode: FormModes.REVIEW,
   children: [
     buildSection({
       id: 'review',
-      title: otherParentApprovalFormMessages.reviewSection,
+      // title: otherParentApprovalFormMessages.reviewSection,
+      title: 'Samþykki',
```

Comment:
Intentionally hard coded to not clash with locale strings in progress and can be integrated